plugins/woocommerce/tests/php/src/Blocks/StoreApi/Routes/CheckoutSessions.php

```
@@ -0,0 +1,573 @@
+		wc_empty_cart();
+		$this->reset_customer_state();
+
+		// Clear all session data early to ensure clean state.
+		if ( WC()->session ) {
+			WC()->session->set( 'chosen_shipping_methods', array() );
+			WC()->session->set( 'store_api_draft_order', 0 );
+			WC()->session->set( 'agentic_draft_order_id', null );
+		}
+
+		// Enable the agentic_checkout feature.
+		$features_controller = wc_get_container()->get( FeaturesController::class );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$features_controller'. (undefined)

(UnusedLocalVariable)